docs/pipeline/actions/introduction.mdx

```diff
@@ -33,6 +33,11 @@ Actions are a powerful tool to help you manage your observability data and ensur
 
 Odigos supports a set of common actions grouped into the following categories:
 
+- [Add Cluster Info](/pipeline/actions/addclusterinfo) - Add static cluster-scoped attributes to your data.
+- [Delete Attribute](/pipeline/actions/deleteattribute) - Delete attributes by name from your data.
+- [Rename Attribute](/pipeline/actions/rename-attribute) - Rename attributes by name in your data.
+- [Probabilistic Sampler](/pipeline/actions/probabilisticsampler) - Add a random sample of your data based on a specified probability.
+- [Pii Masking](/pipeline/actions/piimasking) - Redact attributes by kind of attribute.
```

Comment:
I think this shouldn't be here, as it is already described in [Attributes Actions](/pipeline/actions/attributes/introduction) 2 lines below. Probably merge issue